pkg/domain/domain.go

```diff
@@ -2599,9 +2600,36 @@ func (do *Domain) loadStatsWorker() {
 			if err != nil {
 				logutil.BgLogger().Debug("update stats info failed", zap.Error(err))
 			}
+		case <-do.exit:
+			return
+		}
+	}
+}
+
+func (do *Domain) asyncLoad() {
```

Comment:
How about renaming it to a more specified name like asyncLoadHistogram? We need to show these 2 keywords "async" and "historam"

Reply:
make sense, I have updated it.